scripts/e2e-test.sh

```diff
@@ -3,18 +3,65 @@
 set -eux
 
 SCRIPTDIR=$(dirname "$(realpath "$0")")
-REGISTRY=$1
 TESTS="install basic_volume_io"
+DEVICE=
+REGISTRY=
+TAG=
 
-# TODO: Add proper argument parser
-if [ -z "$REGISTRY" ]; then
-  echo "Missing parameter registry"
+help() {
+  cat <<EOF
+Usage: $(basename $0) [OPTIONS]
+
+Options:
+  --device <path>           Device path to use for storage pools.
+  --registry <host[:port]>  Registry to pull the mayastor images from.
+  --tag <name>              Docker image tag of mayastor images (default "ci")
+
+Examples:
+  $(basename $0) --registry 127.0.0.1:5000 --tag a80ce0c
+EOF
+}
+
+# Parse arguments
+while [ "$#" -gt 0 ]; do
+  case $1 in
```

Comment:
Please _always_ (there are exceptions but IMHO it's not worth using them, I'd just quote everything) quote shell variables. Operator typos and/or spaces can have serious unpredictable consequences: https://github.com/MrMEEE/bumblebee-Old-and-abbandoned/issues/123 (similar bug was in some ubuntu pkg, I cannot find it now, but it was because of space in variable) We need the shellcheck ci :smile: but please run it for the time being manually.

Reply:
👍